php/class-customize-snapshot-manager.php

```diff
@@ -104,6 +104,8 @@ function hooks() {
 		add_action( 'admin_bar_menu', array( $this, 'remove_all_non_snapshot_admin_bar_links' ), 100000 );
 		add_action( 'wp_before_admin_bar_render', array( $this, 'print_admin_bar_styles' ) );
 		add_filter( 'removable_query_args', array( $this, 'filter_removable_query_args' ) );
+		add_filter( 'wp_save_post_revision_post_has_changed', array( $this, '_filter_revision_post_has_changed' ), 20, 3 );
+		add_filter( 'save_post_customize_changeset', array( $this, 'create_initial_changeset_revision' ) );
```

Comment:
@westonruter I think `save_post_customize_changeset ` would be an action and not filter?
https://github.com/WordPress/WordPress/blob/b0bcea2a18f095e2d5b64554782591eacf492c21/wp-includes/post.php#L3494